ouroboros-network/src/Ouroboros/Network/AnchoredFragment.hs

```diff
@@ -709,3 +713,41 @@ mapAnchoredFragment ::
   -> AnchoredFragment block1
   -> AnchoredFragment block2
 mapAnchoredFragment = bimap castAnchor
+
```

Comment:
I think these new definitions belong in a module that is more directly related to BlockFetch and/or ChainSync. This module is lower-level than that, from my current perspective.

Reply:
Moved to `Network/AnchoredFragment/Completeness.hs`